core/src/main/java/com/hcl/ecomm/core/servlets/UpdateCartItems.java

```diff
@@ -29,11 +29,12 @@
     protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response)
             throws ServletException, IOException {
         String  cartItemResponse = "";
+        String customerToken = request.getHeader("CustomerToken");
```

Comment:
@DeepaliAgrwl : why are we getting customer token from request header ? where is that saved in UI ?

Reply:
@rishabh-behl  this token is provided from the ajax call when the call is placed from ui.
Like this:
        xhttp.open("POST", "/bin/hclecomm/shipinfo", true);
	xhttp.setRequestHeader("Content-Type", "application/json;charset=UTF-8");
    	xhttp.setRequestHeader("CustomerToken", custToken);